project_template/webpack.config.js

```diff
@@ -115,31 +113,28 @@ module.exports = {
       { test: /\.yml$/, loader: "json-loader!yaml-loader" },
       {
         test: /\.scss$/,
-        loader: isProd
-          ? extractOtherCss.extract({
-              use: ["css-loader?modules", postcssLoader, "sass-loader"]
-            })
-          : [
-              "style-loader",
-              {
-                loader: "css-loader",
-                options: {
-                  modules: true,
-                  localIdentName: "[path][name]__[local]--[hash:base64:5]"
-                }
-              },
-              postcssLoader,
-              "sass-loader"
-            ],
+        use: [
+          isProd ? MiniCssExtractPlugin.loader : "style-loader",
```

Comment:
I noticed the produced CSS is not minified. According to the [docs](https://github.com/webpack-contrib/mini-css-extract-plugin#minimizing-for-production) this requires a separate plugin.